utils/deploy_lambda_monitor.py

```
@@ -65,7 +60,7 @@ def zip_lambda(source_path: str, arcname: str) -> bytes:
     path = Path(source_path)
 
     if not path.is_file():
-        raise FileNotFoundError(f"Lambda file not found: {path}")
+        logger.error(f"Lambda file not found: {path}", error_type=FileNotFoundError, indent=2)
```

Ruff (0.11.9)

Undefined name `logger`

(F821)